site/viamguides/src/pi-ai-hat/pi-ai-hat.md

```diff
@@ -0,0 +1,305 @@
+1. Click **Save** in the top right. This may take a moment to apply your configuration changes.
+1. At the bottom of the `sensor-1` panel, expand the **TEST** section to ensure you have configured the sensor properly and see a list of the configured labels with the number of detections refreshed regularly.
+   ![test camera](assets/testSensor.png)
+
+With all the components and services in place, you can move on to creating a live tele-operations dashboard for your machine!
+
+<!-- ------------------------ -->
+
+## Create your traffic dashboard
+Duration: 3
+
+This step walks through how to use the [teleop (or tele-operations) feature](https://docs.viam.com/manage/troubleshoot/teleoperate/custom-interface/) of the Viam app.
+
+### Configure your teleop workspace
+
+1. In [the Viam app](https://app.viam.com/fleet/dashboard) under the **TELEOP** tab, create a new dashboard by clicking **Create workspace**.
+   ![create workspace](assets/teleop-landing.png)
+1. Click on "untitle-workspace" to set the name of your dashboard.
+   ![set dashboard name](assets/teleop-set-name.png)
+1. Click "Select machine" and choose your previously configured machine from the list.
+   ![select active machine](assets/teleop-select-machine.png)
+
+### Configure camera stream widget
+
+1. Click "+ Add widget" and select "Camera stream".
+   ![select camera stream widget](assets/teleop-add-camera.png)
+1. Notice adding this widget creates a panel called `Camera`. Select `camera-1` from the "Camera name" field, keep the "Refresh type" as "Live".
+   ![select camera component](assets/teleop-camera-select.png)
+
+### Configure time series graph widget
+
+1. Click "+ Add widget" and select "Time series".
+   ![select time series graph](assets/teleop-add-graph.png)
+1. Notice adding this widget creates a new panel called `Time series`. Set the "Title" to "Traffic" and "Time range (min)" to 30.
+1. Under the **Lines** section of the panel, select `sensor-1` for the "Resource name", `Readings` for "Capture method", `cars` for "Title", and `readings.car` for "Path".
+   ![set graph line](assets/teleop-graph-lines.png)
+1. Add new lines by click "+ Add line". Create a line for each remaining label captured by the sensor: `bus` and `person`.
+1. Click "Save changes" to see the camera stream and data updating live.
+
+![working teleop dashboard](assets/teleop-dashboard-live.png)
+
+<!-- ------------------------ -->
+## Conclusion And Resources
+Duration: 1
+
+### What you learned
+
+- How to configure and test a device's components using Viam
+- How to use modules from the Viam registry
+- How to use a vision service for object detection
+- How to integrate vision service data into a teleop dashboard
+
+### Building advanced scenarios with Viam and the Raspberry Pi AI HAT+
+
+At this point, you have created an edge device that can perform real-time object detection and monitor it remotely from anywhere! You can keep building on this project with additional features:
```

Comment:
Either here, or when configuring the vision service, link to the Other objects that can be detected - is that `hailo-rt` docs? 

I don't want to detect traffic, I want to detect squirrels (or people).

Reply:
I added some details about available labels, other models, and retraining.